### DISCUSSION TITLE
Alerting: Reduce does not support Median like Classic Conditions do

### DISCUSSION THREAD

**Discussion by evan-chaney** [grafana/grafana#73472] [Alerting]
### What happened?

There is no option for median in the reduce expression.

### What did you expect to happen?

I expected it to have all the options that classic condition does since that is what replaces it.

### Did this work before?

Yes, in Grafana 6

### How do we reproduce it?

1. Try to get the median of a timeseries in an alert

### Environment (with versions)?

Grafana: 9.5.1
OS: CentOS 7
Browser: Google Chrome 114.0.5735.198

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

**Comment by armandgrillet** [comment]
Medians are useful for statistics but less for observability which is why it is not available yet, moving this to a discussion as this is a feature request.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82572. Feel free to continue the discussion around this there. Thank you!
